crates/sui-surfer/tests/move_building_blocks/sources/objects.move

```diff
@@ -49,23 +49,35 @@ module move_building_blocks::objects {
         }
     }
 
-    public fun wrap_child(object: &mut Object, child: Child) {
-        unwrap_child(object);
+    public fun create_and_wrap_child(object: &mut Object, ctx: &mut TxContext) {
+        let child = new_child(ctx);
+        wrap_child(object, child, ctx);
+    }
+
+    public fun wrap_child(object: &mut Object, child: Child, ctx: &TxContext) {
+        unwrap_child(object, ctx);
```

Comment:
Do we want a version of this fun that does `unwrap_and_delete_child` instead? so we have `wrapped` and `unwrapped_then_deleted` in the same txn